tsp-cli-client

```diff
@@ -310,17 +320,69 @@ if __name__ == "__main__":
                 print("Trace UUID is missing")
                 sys.exit(1)
 
+        if (options.get_xy_tree):
+            if options.uuid is not None:
+
+                output_descriptor = get_descriptor(
+                    options.uuid, options.get_xy_tree)
+                if (output_descriptor is None):
+                    sys.exit(1)
+
+                response = tsp_client.fetch_xy_tree(
+                    options.uuid, options.get_xy_tree)
+                if response.status_code == 200:
+                    tree = response.model.model
+                    if tree is None:
+                        print("Tree had no model; retry?")
+                        sys.exit(1)
+
+                    if (output_descriptor.type == "DATA_TREE"):
+                        treeModel = TreeModel(tree.entries, tree.descriptors)
+                    else:
+                        treeModel = TreeModel(tree.entries)
+                    treeModel.print()
+                    sys.exit(0)
+                else:
+                    sys.exit(1)
+            else:
+                print("Trace UUID is missing")
+                sys.exit(1)
+
+        if (options.get_xy):
+            if not options.items or not options.times:
+                print("Provide requested --items and --times for the XY data")
+                sys.exit(1)
+
+            if options.uuid is not None:
+                output_descriptor = get_descriptor(
```

Comment:
Not sure that we need it because nothing is done with the returned descriptor.

Reply:
Done (fixed in the latest patch update that I pushed); nice catch, thanks!